DB/Toys/Shadowlands.lua

```diff
@@ -487,7 +487,7 @@ local shadowlandsToys = {
 		},
 		itemId = 187075,
 		chance = 50,
-		requiredDungeons = { [CONSTANTS.LFG_DUNGEON_IDS.TORGHAST_LAYER] = true },
+		requiredDungeons = { [CONSTANTS.LFG_DUNGEON_IDS] = true },
```

Comment:
And this, I believe, inserts the actual table as the key, not the relevant dungeon IDs?